Expecto.Tests/Tests.fs

```diff
@@ -131,7 +131,7 @@ let tests =
     testList "sumTestResults" [
       let sumTestResultsTests =
         let dummyTest = {
-          name = String.Empty
+          name = [String.Empty]
```

Comment:
Should be empty array

Reply:
After refactoring String.Empty will still be here.